tap_zendesk/__init__.py

```diff
@@ -190,10 +191,14 @@ def get_session(config):
 def main():
     parsed_args = singer.utils.parse_args(REQUIRED_CONFIG_KEYS)
 
+    # Set request timeout to config param `request_timeout` value.
+    # If value is 0,"0","" or not passed then it set default to 300 seconds.
+    config_request_timeout = parsed_args.config.get('request_timeout')
+    request_timeout = config_request_timeout and float(config_request_timeout) or REQUEST_TIMEOUT # pylint: disable=consider-using-ternary
```

Comment:
@prijendev Please update this condition as follows for better visibility and understanding
`request_timeout = (config_request_timeout and float(config_request_timeout)) or REQUEST_TIMEOUT`

Reply:
Updated